src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -321,12 +314,13 @@ namespace AirflowNetworkSolver {
             }
             gio::write(Unit11, Format_900) << 0;
             for (i = 1; i <= NetworkNumOfLinks; ++i) {
-                gio::write(Unit11, Format_910) << i << AirflowNetworkLinkageData(i).NodeNums(1) << AirflowNetworkLinkageData(i).NodeHeights(1)
-                                               << AirflowNetworkLinkageData(i).NodeNums(2) << AirflowNetworkLinkageData(i).NodeHeights(2)
+                gio::write(Unit11, Format_910) << i << AirflowNetworkLinkageData(i).NodeNums[0] << AirflowNetworkLinkageData(i).NodeHeights[0]
```

Comment:
Do we have a plan to make array index starting from 0, instead of 1?

Reply:
Yes, removal of all the Objexx arrays is part of the refactoring plan.